src/lib/clients/partners.ts

```diff
@@ -2,19 +2,25 @@ import {
   PartnersClient,
   PartnersClientConfig,
 } from "@stedi/sdk-client-partners";
-import { DEFAULT_SDK_CLIENT_PROPS } from "../constants.js";
+import { requiredEnvVar } from "../environment.js";
+import { NodeHttpHandler } from "@smithy/node-http-handler";
 
 let _partnersClient: PartnersClient | undefined;
 
 export const partnersClient = () => {
   if (_partnersClient === undefined) {
     const config: PartnersClientConfig = {
-      ...DEFAULT_SDK_CLIENT_PROPS,
+      apiKey: requiredEnvVar("STEDI_API_KEY"),
+      region: "us",
+      maxAttempts: 5,
+      requestHandler: new NodeHttpHandler({
```

Comment:
had to switch from the shared `DEFAULT_SDK_PROPS` because the newer SDKs use the `NodeHttpHandler` from the `@smithy` module, not the `@aws` module